webapp/components/notifications/NotificationMenu/NotificationMenu.vue

```diff
@@ -46,24 +48,53 @@ export default {
           variables,
         })
         if (!(markAsRead && markAsRead.read === true)) return
-        this.notifications = this.notifications.map(n => {
-          return n.from.id === markAsRead.from.id ? markAsRead : n
+        this.displayedNotifications = this.displayedNotifications.map(n => {
+          return this.equalNotification(n, markAsRead) ? markAsRead : n
         })
       } catch (err) {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-23T11:03:06Z" title="Monday, September 23rd 2019, 1:03:06 pm +02:00">Sep 23, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -35,7 +49,39 @@ export default {
   props: {
     placement: { type: String },
   },
+  created() {
+    setInterval(this.updateNotifications, REFRESH_MILLISEC)
+  },
+  destroyed() {
+    clearInterval(this.updateNotifications)
+  },
```


![scared](https://media.giphy.com/media/YQA1cC40JlhB2MScI8/giphy.gif)

Please use `pollInterval` by `vue-apollo` and don't re-invent the wheel! See: https://vue-apollo.netlify.com/api/smart-query.html#options

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-23T16:09:49Z" title="Monday, September 23rd 2019, 6:09:49 pm +02:00">Sep 23, 2019</time>_

---


Yeah !!!